paper.md

```diff
@@ -45,7 +45,7 @@ bibliography: paper.bib
 
 # Summary
 
-Groove rolling is one of the main process routes for the production of metal long products, such as bars, beams, wires and rods.
+Groove rolling is one of the main process routes for the production of long metal products, such as bars, beams, wires, and rods.
```

Comment:
So semi-finsihed metal products can be divided into long and flat products. So we would stick with "metal long products"

Reply:
That's fine, if you prefer. I would refer to those as "long metal products" and "flat metal products", since it makes more sense to me to keep the repeating modifier "metal" closer to "products" and place the differentiating modifier first, but if this is the usual convention in your field, that's fine.